actioncable/lib/action_cable/connection/client_socket.rb

```diff
@@ -72,6 +73,7 @@ def rack_response
       def write(data)
         @stream.write(data)
       rescue => e
+        @logger.error "#{e.class} occurred while trying to write to the Action Cable socket: #{e.message}
         emit_error e.message
```

Comment:
`emit_error` already passes it on to the event target. `on_error` seems a better place to handle this.